try {
    arrayConcat(x, array2)
} catch (error) {
    console.log("error")
    
}